src/providers/ogr/qgsogrprovider.cpp

```diff
@@ -1534,6 +1534,12 @@ bool QgsOgrProvider::changeAttributeValues( const QgsChangedAttributesMap &attr_
 
   setRelevantFields( ogrLayer, true, attributeIndexes() );
 
+  bool hasTransaction = OGR_L_TestCapability( ogrLayer, OLCTransactions ) == TRUE;
```

Comment:
Ok, works for me.
Code always looks a bit ancient to me when I see things like this :stuck_out_tongue: